File: a2z/sorting/selection.py
def bubbleSort(arr):
    for i in range(0,len(arr)-1):
        mini = i
        for j in range(i+1,len(arr)):
            if arr[j] < arr[mini]:
                mini = j
        arr[i],arr[mini]=arr[mini],arr[i]


arr = [3,4,5,6,2,0,5,6,7,8,2,3,4,6,7]
bubbleSort(arr)
print(arr)

File: LUV/sorting/selection.cpp
// Find the minimum element in unsorted sublist and place it at the end of sorted sublist
// (at the end of sorted sublist because we have already placed the mini. element at its right position)

#include<bits/stdc++.h>
using namespace std;

void selection(int arr[], int n){
    for(int i=0;i < n;++i){
        int minInd = i;
        for(int j=i+1;j<n;++j){
            if(arr[j] < arr[minInd])
                minInd = j;
        }
        swap(arr[i],arr[minInd]);
    }
}

int main(){
    int n;
    cin >> n;
    int arr[n];
    for(int i=0;i<n;++i) 
        cin >> arr[i];
    
    selection(arr,n);
    for(int i=0;i<n;++i) 
        cout << arr[i] << " ";
}